invenio_administration/assets/semantic-ui/translations/invenio_administration/messages/_generatedTranslations.js

```diff
@@ -1,7 +1,66 @@
-// This is an auto generated file to import and export all available translations.
-// Changes should not be checked into version control.
+// AUTO-GENERATED FILE - DO NOT EDIT MANUALLY
+// This file exports all available translations for i18next
+
+import TRANSLATE_AR from "./ar/translations.json";
```

Comment:
do we need to add this file to the git repository? i thought it will be created on the release step anyway?